knowledge_base/python_wheel_poetry/databricks.yml

```diff
@@ -0,0 +1,34 @@
+bundle:
+  name: python_wheel_poetry
+
+workspace:
+  host: <PLEASE ENTER YOUR DATABRICKS WORKSPACE HOST HERE>
+
+artifacts:
+  default:
+    type: whl
+    build: poetry build
+
+resources:
```

Comment:
I like how this example is minimal, but could we still use a `resources` dir to encourage uniformity?

Reply:
Done.